.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 22.12.0
+    hooks:
+    - id: black
+      language_version: python3.10
```

Comment:
The hard-coded python version caused an error for me on Python 3.8. Worked after I manually changed this to 3.8

Reply:
Not sure if it can be detected somehow? Or maybe we can get away without specifying the language version?

Reply:
good catch, I just removed the language_version entirely and still seems to work. could you try again?

Reply:
Hmm, it no longer does anything, just commits. When I add `language_version` back, then it works as before.

Reply:
weird. it worked for me. I think `default_language_version` is optional, according to [the docs](https://pre-commit.com/#top_level-default_language_version).  I just tested again with a clean install. maybe you need to do `pre-commit install` again?

Also, I added a custom message to the tool telling the user that if the pre-commit fails, they'll need to add and commit again. I don't think there was a way to trigger this on failure, see [this discussion](https://github.com/pre-commit/pre-commit/issues/1565) for more details.